Feature/biblioteca/reservas

MarcusMoraesOliveira opened the pull request:
# [Grupo 2] Manutenção de reservas

* Operações de inserir,deletar,emprestar e alterar
* tratamento de Exceções e erros

MarcusMoraesOliveira committed:
Pagina de reservas

Thileps commented on app/src/java/app/biblioteca/reservas/ErroMain.java:
```diff
@@ -0,0 +1,54 @@
+	public void start(Stage stage) throws Exception {
+		System.out.println("FOI");
+		Parent root = FXMLLoader.load(getClass().getResource("/app/biblioteca/reservas/Erro.fxml"));
+		System.out.println("FOI");
```
Acho q vc esqueceu de remover os dois System.out.pritnln

Thileps reviewed (commented):
Seu código tá dando erro aqui no ControllerEditar.java e no Reservas.java.

isabellaCE committed:
Erros no ControllerEditar.java e no Reservas.java resolvidos e remoção dos System.out.pritnln
